compiler/p/codegen/OMRCodeGenerator.cpp

```diff
@@ -1755,8 +1760,9 @@ OMR::Power::CodeGenerator::freeAndResetTransientLongs()
 
 
 
-bool OMR::Power::CodeGenerator::getSupportsOpCodeForAutoSIMD(TR::ILOpCode opcode, TR::DataType dt)
+bool OMR::Power::CodeGenerator::getSupportsOpCodeForAutoSIMD(TR::ILOpCode opcode, TR::DataType dt, int32_t length)
```

Comment:
Shouldn't the `length` parameter be a `TR::VectorLength` ?  That's how you defined it elsewhere.

Reply:
That was already changed.